docs/current/spies.md

```diff
@@ -399,7 +399,7 @@ Returns the passed format string with the following replacements performed:
     <dt><code>%t</code></dt>
     <dd>a comma-delimited list of <code>this</code> values the spy was called on</dd>
 
-    <dt><code>%<var>n</var></code></dt>
+    <dt><code>%</code><var><code>n</code></var></dt>
```

Comment:
If I read the HTML spec correctly, then `<var>` is a perfectly valid child of `<code>`.

https://www.w3.org/TR/html5/text-level-semantics.html#the-code-element
https://www.w3.org/TR/html5/dom.html#phrasing-content-1

Do you agree? Is this change necessary?